### PR TITLE
Add translate pt_BR files for Hop Project

### PR DESCRIPTION
**Pull request by marciojv**
Hi Matt , 
how about hop in portugues from Brazil?

### CODE REVIEW

**Comment by hansva** [comment]
Hi @marciojv, Thanks for the contribution all extra languages are very welcome!
But I currently can't accept your pull request.. we want to move to ASF in the near future and references to persons or authors in any form are not allowed under the ASF guidelines. All code belongs to the project and not the contributors.

If you remove the first line, I will happily merge this PR

File: archive-transformation-steps/src/main/resources/engine/steps/delay/messages/messages_pt_BR.properties [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#File translate by Marcio Junior Vieira (Ambiente Livre) for package 'org.apache.hop.trans.steps.Delay' in locale 'pt_BR'
```
Comment (hansva) [inline archive-transformation-steps/src/main/resources/engine/steps/delay/messages/messages_pt_BR.properties]: remove author reference, not allowed under ASF guidelines

File: ui/src/main/resources/org/apache/hop/ui/i18n/editor/messages/messages_pt_BR.properties [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#File translate by Marcio Junior Vieira (Ambiente Livre) for package 'org.apache.hop.ui.i18n.editor' in locale 'pt_BR'
```
Comment (hansva) [inline ui/src/main/resources/org/apache/hop/ui/i18n/editor/messages/messages_pt_BR.properties]: Remove author, not allowed under ASF guidelines

**Comment by marciojv** [comment]
Hi @hansva see is ok now! 
Go Open Source! Go Apache Hop!

**Comment by hansva** [comment]
@marciojv, Thanks for the update! merged to master! We created a ticket in Jira https://project-hop.atlassian.net/browse/HOP-136 and would love to have as many languages as possible in Hop.
Matt also included an older project back in hop to help with translations, Explanation on how to use this will be added to this readme and website soon

**Comment by marciojv** [comment]
Thanks, My friend @eduardojosedaniel  too go help in this Job!

**Pull request merged**